remotes/docker/authorizer.go

```diff
@@ -153,6 +153,11 @@ func (a *dockerAuthorizer) AddResponses(ctx context.Context, responses []*http.R
 				return err
 			}
 
+			if c.Parameters["error"] == "invalid_token" {
```

Comment:
is it from special platform?

Reply:
> is it from special platform?

Not sure if this is the generic err message protocol for all platforms, but I verified dockerhub and alibaba container registry both gave this errcode when invalid or expired token is provided in header.

Reply:
Seems like this could be slotted into `invlaidAuthorization` here: https://github.com/containerd/containerd/blob/296d67e5f33d22fbf49f73cc703a608f70688474/remotes/docker/authorizer.go#L341